.github/workflows/e2e.yaml

```diff
@@ -434,6 +436,7 @@ jobs:
           working-directory: ./${{ inputs.e2eTestsRepo }}
         env:
           CYPRESS_ORGANIZATION_ID: ${{steps.create_org_for_tests.outputs.organizationId}}
```

Comment:
You could imagine a future improvement where we randomly generate the password in the demo org creation, and then pass it to the E2R like we do the organizationId, but this is fine for now too!